### PR TITLE
Settings menu

### PR DESCRIPTION
**Pull request by m-lin1**

### CODE REVIEW

File: Scripts/Level_Components/Level_Elements/SettingsMenu.gd [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+func reset_brightness():
+	brightnessSlider.value = bStartVal
+	
+
+func continue_pressed():
+	get_tree().paused = false
+	visible = false
+	
+
+func exit_level_pressed():
```
[the remark this answers is not in the document]
Comment (he-larious) [reply Scripts/Level_Components/Level_Elements/SettingsMenu.gd:87]: I highlight the wrong things to comment on but ykwim

[the remark this answers is not in the document]
Comment (m-lin1) [reply Scripts/Level_Components/Level_Elements/SettingsMenu.gd:87]: As discussed on Discord: Add a Tutorial Level Select & Tutorial Stage in Stage Select

File: Scripts/Singletons/SceneSwapper.gd [hunk 2]
```diff
@@ -33,7 +34,10 @@ func change_scene(scene_name):
 func _deferred_change_scene(scene_name):
 	# It is now safe to remove the current scene
 	current_scene.free()
```
[the remark this answers is not in the document]
Comment (m-lin1) [reply Scripts/Singletons/SceneSwapper.gd:37]: set current_scene again in change_scene() function to fix this

**Review by he-larious — COMMENTED** [review]
Everything is good to go but I think I'm gonna wait til I do the Tutorial Select page before we merge this PR in so we don't have broken code in main.

**Review by he-larious — APPROVED** [review]

**Pull request merged**